### DISCUSSION TITLE
Font properties extraction for specific text

### DISCUSSION THREAD

**Discussion by meghanaviyyapu** [pymupdf/PyMuPDF#3985] [Looking for help]
Could you let me know if it is possible to extract font properties of a specific text from a PDF? I have used span wise extraction of font properties but if I need font properties of a specific text, is it possible to get them using PyMuPDF?

**Comment by JorjMcKie** [comment]
Why did looking at spans not work?

**Reply by meghanaviyyapu** [reply]
It works when the specific text matches with span["text"] but in some cases, the specific text needed may not match with the span["text"]. 
What should be the approach when specific text does not match with span["text"]?

**Reply by JorjMcKie** [reply]
search for the text in question, then take (one of) the returned rectangles and again execute `get_text("dict", clip=rect)`. At least parts of the text should be in the returned spans

**Reply by JorjMcKie** [reply]
If the search function returns multiple rectangles: never mind! Just take any of them, e.g. the first one.
If you make no error, all parts of the text should use the same font, right? So checking any of the search rectangles should deliver the desired result.

**Reply by meghanaviyyapu** [reply]
Ok thanks a lot for clarifying. Would try the same
